src/app/ui/icon/icon.component.html

```diff
@@ -0,0 +1,14 @@
+<img
+  [src]="'../../../assets/images/icons/' + name + '.svg'"
```

Comment:
Is this working the way you want it to? I would prefer to have those .svg files declared as assets

Reply:
Yes, it is working the way I want it to.

Reply:
Ok